great_expectations/rule_based_profiler/data_assistant/data_assistant_dispatcher.py

```diff
@@ -105,3 +105,16 @@ def get_data_assistant_impl(
             return None
         name = name.lower()
         return cls._registered_data_assistants.get(name)
+
+    def __dir__(self):
```

Comment:
This is the perfect place to put this! 🙇🏻

Reply:
Agreed 🙇🏼